loom/test/test_infer.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import numpy as np
+import json
+
+import loom.tasks
+
+def _learn_seeded_model(seed):
+    """Helper function to learn the exact same taxi model with different seeds"""
+    config = {'schedule': {'extra_passes': 1.0}, 'seed':seed}
+    name = 'seeding-test'
+    loom.tasks.ingest(name, 'synth-schema.json', 'synth.csv', debug=True)
+    loom.tasks.infer(name, sample_count=1, config=config, debug=True)
+    with loom.tasks.query(name) as server:
+        dependencies = server.relate(['a', 'b'])
+    return dependencies
+
+def test_seeding():
+    """Test if we can seed learning"""
+    schema = {'a':'nich', 'b': 'nich'}
+    with open('synth-schema.json', 'w') as outfile:
+        json.dump(schema, outfile)
+    df = pd.DataFrame({
+        'a':np.random.normal(0, 1, size=10),
+        'b':np.random.normal(0, 1, size=10)
+    })
```

Comment:
It would probably be a good idea to control the stochasticity of the test rather than use global randomness, which can lead to excessive stochastic or confusing failures.

One option is to migrate the [@stochastic decorator](https://github.com/probcomp/bayeslite/blob/master/tests/stochastic.py) from bayeslite into here, which should be easy.

A second (easier) option is to the hard-code the seed or parametrize the test on 10 different fixed seeds, which will provide less coverage but more robustness.

Reply:
Loom does a bunch of things with nose. I can look into this. I suspect nose is going to make things tricky.